### PR TITLE
Reformat admonitions

### PR DESCRIPTION
**Pull request by gsteel**
|    Q          |   A
|-------------- | ------
| Documentation | yes

### Description

Removes unnecessary error suppression

### CODE REVIEW

**Comment by gsteel** [comment]
@froschdesign
I couldn't see "MISSING" as an option at https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types so I went for "NOTE" ?? Correct?

File: docs/book/find-feeds.md [hunk 1]
```diff
@@ -8,8 +8,7 @@ feeds referenced by a web page with one method call:
 $feedLinks = Laminas\Feed\Reader\Reader::findFeedLinks('http://www.example.com/news.html');
 ```
 
-<!-- markdownlint-disable-next-line heading-increment -->
-> ### Finding feed links requires an HTTP client
+> NOTE: Finding feed links requires an HTTP client
```
Comment (froschdesign) [inline docs/book/find-feeds.md]:
```suggestion
> MISSING: **Finding Feed Links Requires an HTTP Client**
```

File: docs/book/importing.md [hunk 2]
```diff
@@ -8,8 +8,7 @@ to consume it:
 $feed = Laminas\Feed\Reader\Reader::import('http://feeds.example.com/feedName');
 ```
 
-<!-- markdownlint-disable-next-line heading-increment -->
-> ### Importing requires an HTTP client
+> NOTE: Importing requires an HTTP client
```
Comment (froschdesign) [inline docs/book/importing.md]:
```suggestion
> MISSING: **Importing Requires an HTTP Client**
```

File: docs/book/intro.md [hunk 3]
```diff
@@ -18,8 +18,7 @@ In the example below, we demonstrate a simple use case of retrieving an RSS feed
 and saving relevant portions of the feed data to a simple PHP array, which could
 then be used for printing the data, storing to a database, etc.
 
-<!-- markdownlint-disable-next-line heading-increment -->
-> ### RSS optional properties
+> NOTE: **RSS optional properties**
```
Comment (froschdesign) [inline docs/book/intro.md]:
```suggestion
> NOTE: **RSS Optional Properties**
```

**Review by froschdesign — CHANGES_REQUESTED** [review]

**Comment by froschdesign** [comment]
@gsteel 
> @froschdesign I couldn't see "MISSING" as an option at https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types so I went for "NOTE" ?? Correct?

`MISSING` is in the list: "failure, fail, missing"

See also:

> Custom look[#](https://oprypin.github.io/markdown-callouts/#custom-look)
> As [mentioned](https://oprypin.github.io/markdown-callouts/#output), styling is handled through CSS, not from the extension itself. And in CSS, you can indeed completely re-define the look for each particular keyword. **Adding a new keyword requires no action, other than possibly adding CSS for it**.

https://oprypin.github.io/markdown-callouts/#custom-look

https://github.com/laminas/laminas.github.io/blob/3e2faddba0e7d00de92b0b77cb655c4794a742b3/asset/sass/_settings.scss#L18-L46

**Comment by Ocramius** [comment]
🚢

**Pull request merged**